packages/react/src/date-range-picker-next/DateRangePickerNext.tsx

```diff
@@ -511,3 +559,15 @@ export function useDateRangePickerNextIds(idProp?: string) {
 	const toId = `date-range-picker-${autoId}-to`;
 	return { fieldsetId, fromId, hintId, messageId, toId };
 }
+
+// Attempts focus on target and returns true if successful
+function focusDay(queryTarget: string) {
+	const day = document.querySelector(queryTarget);
+	if (!day) return false;
+
+	if (day instanceof HTMLElement && day.focus) {
+		day.focus();
+		return true;
+	}
+	return false;
```

Comment:
`focus` exists on every elememt, right? So now we return early, can we simplify it to this?
```suggestion
	(day as HTMLElement).focus();
	return true;
```

Reply:
`document.querySelector` returns an `Element` type and not `HTMLElement`. I don't think it will return anything that might not have a focus?